CoreTweet.Shared/Objects/Events.cs

```diff
@@ -0,0 +1,201 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CoreTweet.Core;
+using Newtonsoft.Json;
+
+namespace CoreTweet
+{
+    public class Event : CoreBase
+    {
+        [JsonProperty("for_user_id")]
+        public string ForUserId { get; set; }
+
+        [JsonProperty("tweet_create_events")]
+        public Status[] TweetCreateEvents { get; set; }
+
+        [JsonProperty("favorite_events")]
+        public FavoriteEvent[] FavoriteEvents { get; set; }
+
+        [JsonProperty("follow_events")]
+        public SourceTargetEvent<User, User> FollowEvents { get; set; }
+
+        [JsonProperty("block_events")]
+        public SourceTargetEvent<User, User> BlockEvents { get; set; }
+
+        [JsonProperty("mute_events")]
+        public SourceTargetEvent<User, User> MuteEvents { get; set; }
+
+        [JsonProperty("user_event")]
+        public UserEvent UserEvent { get; set; }
+
+        [JsonProperty("direct_message_events")]
+        public DirectMessageEvent[] DirectMessageEvents { get; set; }
+
+        [JsonProperty("direct_message_indicate_typing_events")]
+        public DirectMessageIndicateTypingEvent[] DirectMessageIndicateTypingEvents { get; set; }
+
+        [JsonProperty("direct_message_mark_read_events")]
+        public DirectMessageMarkReadEvent[] DirectMessageMarkReadEvents { get; set; }
+
+        [JsonProperty("tweet_delete_events")]
+        public TweetDeleteEvent[] TweetDeleteEvents { get; set; }
+
+        [JsonProperty("apps")]
+        public IDictionary<string, App> Apps { get; set; }
+
+        [JsonProperty("users")]
+        public IDictionary<string, User> Users { get; set; }
+    }
+
+    public class FavoriteEvent : CoreBase
+    {
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("created_at")]
+        [JsonConverter(typeof(DateTimeOffsetConverter))]
+        public DateTimeOffset CreatedAt { get; set; }
+
+        [JsonProperty("timestamp_ms")]
+        [JsonConverter(typeof(TimestampConverter))]
+        public DateTimeOffset Timestamp { get; set; }
+
+        [JsonProperty("favorited_status")]
+        public Status FavoritedStatus { get; set; }
+
+        [JsonProperty("user")]
+        public User User { get; set; }
+    }
+
+    public class SourceTargetEvent<TSource, TTarget> : CoreBase
+    {
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        [JsonProperty("created_timestamp")]
+        [JsonConverter(typeof(TimestampConverter))]
+        public DateTimeOffset Timestamp { get; set; }
```

Comment:
Why does the name of this property need to be different from the original name, `created_timestamp` ?

Reply:
@azyobuzin Sorry, It's my mistake. I'll fix it later.